ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -447,7 +457,7 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
               )}
             {phone &&
```

Comment:
We could restore the previous version that we know it works and remove the method **openPhoneDialer**

```typescrip
{phone &&
              renderInformationRow(I18n.t("services.contactPhone"), phone, () =>
                Linking.openURL(`tel:${phone}`).then(() => 0, () => 0)
              )}
```